crates/iota/src/iota_commands.rs

```diff
@@ -156,6 +156,20 @@ pub enum IotaCommand {
     /// generate the genesis blob, and start the network.
     ///
     /// Note that if you want to start an indexer, Postgres DB is required.
+    ///
+    /// Protocol config parameters can be overridden individually by setting
+    /// environment variables as follows:
+    /// - IOTA_PROTOCOL_CONFIG_OVERRIDE_ENABLE=1
+    /// - Then, to configure an override, use the prefix
+    ///   `IOTA_PROTOCOL_CONFIG_OVERRIDE_` along with the parameter name. For
+    ///   example, to increase the interval between checkpoint creation to >1/s,
+    ///   you might set:
+    ///   IOTA_PROTOCOL_CONFIG_OVERRIDE_min_checkpoint_interval_ms=1000
+    ///
+    /// Note that protocol config parameters must match between all nodes, or
+    /// the network may break. Changing these values outside of local
+    /// networks is very dangerous.
+    #[clap(verbatim_doc_comment)]
```

Comment:
I think we don't use the clap attribute anymore

Reply:
I tested it ... we need it here unfortunately.

Reply:
Works for me if I replace it

Reply:
for me the formatting is messed up then, because `clap`s formatting removes whitespace.

Reply:
I took the liberty of doing the change

Reply:
yeah, fine, thought you meant the specific attribute.